common/supplemental/likelySubtags.xml

```diff
@@ -1036,6 +1046,7 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<likelySubtag from="und_Ahom" to="aho_Ahom_IN"/>		<!--?‧Ahom‧?	➡ Ahom‧Ahom‧India-->
 		<likelySubtag from="und_Arab" to="ar_Arab_EG"/>		<!--?‧Arabic‧?	➡ Arabic‧Arabic‧Egypt-->
 		<likelySubtag from="und_Arab_AF" to="fa_Arab_AF"/>		<!--?‧Arabic‧Afghanistan	➡ Persian‧Arabic‧Afghanistan-->
+		<likelySubtag from="und_Arab_AZ" to="az_Arab_AZ"/>		<!--?‧Arabic‧Azerbaijan	➡ Azerbaijani‧Arabic‧Azerbaijan-->
```

Comment:
@macchiati wrote:
> I think @roozbehp  had some comments about this one, since it would be exceedingly rare to have Azerbaijan written in Arabic script these days.

Reply:
Roozbeh's comment in the [previous attempt](https://github.com/unicode-org/cldr/pull/3998) was about `und_Arab_AZ` mapping to `tly_Arab_AZ` -- he said that was very unlikely and it probably should be `az_Arab_AZ` as you see in this PR. Because of multiple scripts this gets automatically generated so we have to have some value here -- at least in this PR its the right one.